lib/krikri/harvesters/fedora_harvester.rb

```diff
@@ -0,0 +1,158 @@
+require 'uri'
+require 'async_uri_getter'
+
+module Krikri::Harvesters
+  ##
+  # A harvester implementation for Fedora 3
```

Comment:
My guess was it would be better to implement this as a generic Fedora 3 harvester. Happy to recast it as UVa specific if that's better.